otx/algorithms/visual_prompting/adapters/pytorch_lightning/datasets/dataset.py

```diff
@@ -0,0 +1,331 @@
+        """Setup Visual Prompting Data Module.
+
+        Args:
+            stage (Optional[str], optional): train/val/test stages, defaults to None.
+        """
+        if not stage == "predict":
+            self.summary()
+
+        image_size = self.config.image_size
+        if isinstance(image_size, int):
+            image_size = [image_size]
+
+        if stage == "fit" or stage is None:
+            train_otx_dataset = self.dataset.get_subset(Subset.TRAINING)
+            val_otx_dataset = self.dataset.get_subset(Subset.VALIDATION)
+
+            # TODO (sungchul): distinguish between train and val config here
+            train_transform = val_transform = MultipleInputsCompose(
+                [
+                    ResizeLongestSide(target_length=max(image_size)),
+                    Pad(),
+                    transforms.Normalize(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375]),
+                ]
+            )
+
+            self.train_dataset = OTXVIsualPromptingDataset(
+                train_otx_dataset, train_transform, offset_bbox=self.config.offset_bbox
+            )
+            self.val_dataset = OTXVIsualPromptingDataset(val_otx_dataset, val_transform)
+
+        if stage == "test":
+            test_otx_dataset = self.dataset.get_subset(Subset.TESTING)
+            test_transform = MultipleInputsCompose(
+                [
+                    ResizeLongestSide(target_length=max(image_size)),
+                    Pad(),
+                    transforms.Normalize(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375]),
+                ]
+            )
+            self.test_dataset = OTXVIsualPromptingDataset(test_otx_dataset, test_transform)
+
+        if stage == "predict":
+            predict_otx_dataset = self.dataset
+            predict_transform = MultipleInputsCompose(
+                [
+                    ResizeLongestSide(target_length=max(image_size)),
+                    Pad(),
+                    transforms.Normalize(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375]),
+                ]
+            )
+            self.predict_dataset = OTXVIsualPromptingDataset(predict_otx_dataset, predict_transform)
+
+    def summary(self):
+        """Print size of the dataset, number of images."""
+        for subset in [Subset.TRAINING, Subset.VALIDATION, Subset.TESTING]:
+            dataset = self.dataset.get_subset(subset)
+            num_items = len(dataset)
+            logger.info(
+                "'%s' subset size: Total '%d' images.",
+                subset,
+                num_items,
+            )
+
+    def train_dataloader(self) -> Union[DataLoader, List[DataLoader], Dict[str, DataLoader]]:
+        """Train Dataloader.
+
+        Returns:
+            Union[DataLoader, List[DataLoader], Dict[str, DataLoader]]: Train dataloader.
+        """
+        return DataLoader(
+            self.train_dataset,
+            shuffle=False,
```

Comment:
is this okay to set up `shuffle=False` when training?

Reply:
Fixed, but I don't think the results will be much different from before.